### ISSUE TITLE
Early Earthquake Detection Using BNGCNN

### ISSUE THREAD

**Issue by chippen001** [chippen001/Capstone-Resources#6]
### Title
Early Earthquake Detection Using Batch Normalization Graph Convolutional Neural Network (GNGCNN)

### URL
https://www.mdpi.com/2076-3417/12/15/7548#:~:text=CNN%20layers%20aggregate%20features%20from,detection%20from%20seismic%20array%20data 

### Summary 
This article 

### Key Points 

- 

### Citation
Muhammad Atif Bilal, Yanju Ji, Yongzhi Wang, Muhammad Pervez Akhter, and Muhammad Yaqub. Early earthquake detection using batch normalization graph convolutional neural network (bngcnn). Applied Sciences, 12(15), 2022.
